h/static/scripts/widget-controller.js

```diff
@@ -312,9 +347,11 @@ module.exports = function WidgetController(
     streamer.reconnect();
   });
 
-  // When a direct-linked annotation is successfully anchored in the page,
-  // focus and scroll to it
   $rootScope.$on(events.ANNOTATIONS_SYNCED, function (event, tags) {
```

Comment:
In widget-controller, we select a tab in 4 cases:
1. page load (select annotations)
2. on group change (select annotations)
3. on group change/page load to a group which has no annotations (select annotations. Also, the ANNOTATIONS_SYNCED event isn't fired in this case).
4. when a user lands on a direct link (select the tab based on the annotation type)

For cases #1 and #4, after the ANNOTATIONS_SYNCED event is fired, we can tell between anchored and unanchored annotations.